README.md

```diff
@@ -361,6 +361,38 @@ class PetFactory(ModelFactory):
 
 `Use` is merely a semantic abstraction that makes the factory cleaner and simpler to understand.
 
+#### PostGenerated (field)
+
+It allows for post generating fields based on already generated values of other
```

Comment:
line spacing here appears to be with manual breaks. Best to let prettier handle this IMO.